bot/exts/info/tags.py

```diff
@@ -362,8 +292,8 @@ def accessible_tags_in_group(self, group: str, user: discord.Member) -> list[str
             if identifier.group == group and tag.accessible_by(user)
         )
 
-    @tags_group.command(name="get", aliases=("show", "g"), usage="[tag_group] [tag_name]")
-    async def get_command(self, ctx: Context, *, argument_string: Optional[str]) -> bool:
+    @tag_group.command(name="get")
+    async def get_command(self, interaction: discord.Interaction, *, tag_name: Optional[str]) -> bool:
```

Comment:
```suggestion
    async def get_command(self, interaction: discord.Interaction, *, name: Optional[str]) -> bool:
```

Looks a bit neater in the command, since it's obvious it's the tag name so no need to repeat tag. (will require updates in the rest of the function ofc)